python/graphstorm/gconstruct/transform.py

```diff
@@ -989,6 +995,13 @@ def parse_feat_ops(confs):
                                       infer_batch_size=infer_batch_size,
                                       out_dtype=out_dtype)
             elif conf['name'] == 'max_min_norm':
+                # TODO: Not support max_min_norm feature transformation on multiple columns
+                # without explicitly defining max_val and min_val. Otherwise, it is not clear to
```

Comment:
```suggestion
                # without explicitly defining max_val and min_val. Otherwise, the definition of max_val and min_val for each column is unclear.
```